lib/args.js

```diff
@@ -72,8 +73,15 @@ function args(argv) {
       }
     });
 
+  if (opts.hasOwnProperty('url')) {
+    opts.url = url.parse(opts.url);
+    delete opts.url.host;
```

Comment:
This is really terrible, but apparently it's what needs to be done when using the legacy `url`  API, see: https://nodejs.org/dist/latest-v8.x/docs/api/url.html#url_url_format_urlobject

The WHATWG URL API (which is much nicer) is supported in Node 7+, so we would need to add a polyfill or sth in here if we wanted to use it. Don't know if that's worth it.